samples/cpp/benchmark_app/utils.cpp

```diff
@@ -877,6 +877,28 @@ std::vector<std::string> filter_files_by_extensions(const std::vector<std::strin
 std::string parameter_name_to_tensor_name(const std::string& name,
                                           const std::vector<ov::Output<const ov::Node>>& inputs_info,
                                           const std::vector<ov::Output<const ov::Node>>& outputs_info) {
+    
+
+    auto found_past_key = name.find("past_keys");
+    auto found_past_value = name.find("past_values");
+    auto found_past_key_values = name.find("past_key_values");
+    auto found_key = name.find(".key");
+    auto found_value = name.find(".value");
+    auto found_present = name.find("present");
+
+    if (found_past_key != std::string::npos) {
```

Comment:
There's no need to find everything in advance and then run the checks. Instead of looping through the `name` string 6 times you can move those finds into the condition, something like this:
```suggestion
    if (name.find("past_keys") != std::string::npos) {
```